constraint/pkg/client/client_addconstraint_bench_test.go

```diff
@@ -0,0 +1,59 @@
+package client
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	"github.com/open-policy-agent/frameworks/constraint/pkg/client/drivers/local"
+	"github.com/open-policy-agent/frameworks/constraint/pkg/core/templates"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+func makeConstraint(i int, ct *templates.ConstraintTemplate) *unstructured.Unstructured {
```

Comment:
I assume the benchmark overhead of this is minimal? Would it be helpful to build the data structure to have everything except the name and then copy the contents of the pointer and set the name?

Reply:
The overhead is indeed negligible. In a follow-up PR I refactor this and add b.StopTimer() and b.StartTimer() around it to be super-sure.